qiita_db/test/test_util.py

```diff
@@ -616,6 +619,47 @@ def test_check_access_to_analysis_result(self):
 
         self.assertEqual(obs, exp)
 
+    def test_add_message(self):
+        count = get_count('qiita.message') + 1
+        users = [User('shared@foo.bar'), User('admin@foo.bar')]
+        add_message("TEST MESSAGE", users)
+
+        obs = [[x[0], x[1]] for x in User('shared@foo.bar').messages()]
+        exp = [[count, 'TEST MESSAGE'], [1, 'message 1']]
+        self.assertEqual(obs, exp)
+        obs = [[x[0], x[1]] for x in User('admin@foo.bar').messages()]
+        exp = [[count, 'TEST MESSAGE']]
+        self.assertEqual(obs, exp)
+
+    def test_add_system_message(self):
+        count = get_count('qiita.message') + 1
+        add_system_message("SYS MESSAGE", datetime(2015, 8, 5, 19, 41))
+
+        obs = [[x[0], x[1]] for x in User('shared@foo.bar').messages()]
+        exp = [[count, 'SYS MESSAGE'], [1, 'message 1']]
+        self.assertEqual(obs, exp)
+        obs = [[x[0], x[1]] for x in User('admin@foo.bar').messages()]
+        exp = [[count, 'SYS MESSAGE']]
+        self.assertEqual(obs, exp)
+
+        sql = "SELECT expiration from qiita.message WHERE message_id = %s"
+        obs = self.conn_handler.execute_fetchall(sql, [count])
+        exp = [[datetime(2015, 8, 5, 19, 41)]]
+        self.assertEqual(obs, exp)
+
+    def test_clear_system_messages(self):
+        obs = [[x[0], x[1]] for x in User('shared@foo.bar').messages()]
+        exp = [[1, 'message 1']]
+        self.assertEqual(obs, exp)
+
+        add_system_message("SYS MESSAGE", datetime(2015, 8, 5, 19, 41))
+        obs = [[x[0], x[1]] for x in User('shared@foo.bar').messages()]
+        exp = [[1, 'message 1'], [4, 'SYS MESSAGE']]
```

Comment:
BTW, you're missing the `self.assertEqual(obs, exp) here and in the test below... so you're not testing anything right now.